crates/bevy_pbr/src/prepass/mod.rs

```diff
@@ -412,7 +412,9 @@ where
             .mesh_key
             .intersects(MeshPipelineKey::NORMAL_PREPASS | MeshPipelineKey::DEFERRED_PREPASS)
         {
-            vertex_attributes.push(Mesh::ATTRIBUTE_NORMAL.at_shader_location(3));
+            if layout.0.contains(Mesh::ATTRIBUTE_NORMAL) {
+                vertex_attributes.push(Mesh::ATTRIBUTE_NORMAL.at_shader_location(3));
+            }
```

Comment:
Something like this will warn in the case that you are trying to use the default normal prepass without normal attributes

```suggestion
            if layout.0.contains(Mesh::ATTRIBUTE_NORMAL) {
                vertex_attributes.push(Mesh::ATTRIBUTE_NORMAL.at_shader_location(3));
            } else if matches!(M::prepass_fragment_shader(), ShaderRef::Default) {
       			warn!("The default normal prepass expects the mesh to have vertex normal attributes");
          	}
```

Reply:
It won't work on the StandardMaterial though, but that's better than nothing I guess.